SWIFT/day11.swift

```diff
@@ -1,22 +1,18 @@
 import Foundation
 
-var matrix: [[Int]] = [[]]
-for i in 0..<6 {
-    let line: [String] = readLine()!.characters.split { $0 == " "}.map(String.init)
-    matrix.append([]) // Create row
-    for elem in line {
-        matrix[i].append(Int(elem)!)
-    }   
+var matrix = [[Int]]()
+for _ in 0..<6 {
+    matrix.append(readLine()!.characters.split(" ").map(String.init).map{Int($0)!})
```

Comment:
But this maps on arrays in general are used as separate because to make them more readable. Time and Processing complexities are same for both statements